src/utils/redux/userSlice.js

```diff
@@ -5,7 +5,7 @@ import { createSlice } from '@reduxjs/toolkit';
 Request URL: http://localhost:8080/meetingTypes/create <-- this is what you call when you click "Submit" for when the case worker is finished with created the meeting types/preferences
 Json payload (shape):
 {
-    "name":"Training" | "Inspection" | "The other meeting type" | "Move In" | "The last meeting type"
```

Comment:
Based on this payload shape, it looks like Tracy's service might only accept these specific strings for the "name" field. Obviously they're just placeholders that she's using, but we should make sure her service will accept the right types